src/components/Logo/AssetLogo.tsx

```diff
@@ -77,11 +82,22 @@ export default function AssetLogo({
 
   const [src, nextSrc] = useTokenLogoSource(address, chainId, isNative, backupImg)
   const L2Icon = getChainInfo(chainId)?.circleLogoUrl
+  const [imgLoaded, setImgLoaded] = useState(false)
 
   return (
     <LogoContainer style={style}>
       {src ? (
-        <LogoImage {...imageProps} src={src} onError={nextSrc} useBG={true} />
+        <LogoImageWrapper size={size} useBG={!imgLoaded}>
+          <LogoImage
+            {...imageProps}
```

Comment:
nit: I think `imageProps` makes this more confusing - can you remove it and just inline the props?